bigframes/core/compile/scalar_op_compiler.py

```diff
@@ -894,6 +894,16 @@ def array_to_string_op_impl(x: ibis_types.Value, op: ops.ArrayToStringOp):
     return typing.cast(ibis_types.ArrayValue, x).join(op.delimiter)
 
 
+# JSON Ops
+@scalar_op_compiler.register_binary_op(ops.JSONSet, pass_op=True)
+def json_set_op_impl(x: ibis_types.Value, y: ibis_types.Value, op: ops.JSONSet):
+    return vendored_ibis_ops.JSONSet(
+        x,
+        json_value=y,
+        json_path=op.json_path,
+    ).to_expr()
```

Comment:
is it possible to implement through the ibis udf interface, or is the syntax too non-standard? The new custom op will add extra work when upgrading to ibis 9

Reply:
See: https://github.com/googleapis/python-bigquery-dataframes/blob/ec5b06856762e67060642cc106f43f0d202a08bf/bigframes/core/compile/scalar_op_compiler.py#L1462 for an example.

Reply:
yes, it can be implemented through ibis udf interface.